src/python/pants/backend/python/typecheck/pyright/rules.py

```diff
@@ -58,6 +59,10 @@ class PyrightFieldSet(FieldSet):
     resolve: PythonResolveField
     interpreter_constraints: InterpreterConstraintsField
 
+    @classmethod
```

Comment:
Is this still needed? I've seen it on `mypy` but is it on the linters/formatters?

Reply:
nvm.